src/reusedBrowser.ts

```diff
@@ -164,6 +168,10 @@ export class ReusedBrowser implements vscodeTypes.Disposable {
         const range = this._editor.document.validateRange(new this._vscode.Range(start, end));
         this._editor?.edit(async editBuilder => {
           editBuilder.replace(range, params.text);
+          if (this._editor) {
+            const lastLine = this._editor.document.lineCount - 1;
+            this._editor.selections = [new this._vscode.Selection(lastLine, 0, lastLine, 0)];
```

Comment:
Why this? It seems to put the cursor at the end, but I am not sure about the purpose of this.